### PR TITLE
Created horizontal scroll in sponsors section + contact and footer

### PR DESCRIPTION
**Pull request by julesparra**

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -159,3 +159,45 @@
   max-height: 0;
   overflow: hidden;
   transition: max-height 0.1s ease-out; }
+
+.sponsor-img {
+  width: 100px;
+  height: 100px;
+  position: relative;
```
Comment (julesparra) [inline css/main.css:166]: i couldn't achieve a smooth transition from right edge of the screen to the left without making this absolute, but then images overlap obv. thoughts?

File: index.html [hunk 2]
```diff
@@ -303,6 +303,79 @@ <h4>+</h4>
...
+                <img class="sponsor-img" src="./img/w_linkedin.svg" alt="">
+                <img class="sponsor-img" src="./img/w_medium.svg" alt="">
+                <img class="sponsor-img" src="./img/w_twitter.svg" alt="">
+                <img class="sponsor-img" src="./img/w_youtube.svg" alt="">
+            </div>
+          </div>
+      </div>
+    </div>
+        
+    <div class="contact">
```
Comment (apanpanpan) [inline index.html:329]: omg thank u for doing the contact and the footer!!!!

File: index.html [hunk 3]
```diff
@@ -303,6 +303,79 @@ <h4>+</h4>
           </div>
       </div>
     </div>
+        
+    <div class="sponsors" style="background-color: grey;">
+      <div class="container" style="padding: 3%; padding-bottom: 10%">
+          <div class="row">
+            <div class="col-sm">
```
Comment (apanpanpan) [inline index.html:310]: nit: i dont think we need to add a row/col for this header - same goes for the FAQ header

File: index.html [hunk 4]
```diff
@@ -303,6 +303,79 @@ <h4>+</h4>
...
+    <div class="contact">
+      <div class="container" style="padding: 1%;">
+        <div class="contact-header">
+          <h4>Contact Us</h4>
+        </div>
+        <div class="contact-content">
+            <p>Reach out to us at <a href="mailto:hello@gotechnica.org">hello@gotechnica.org</a></p>
+        </div>
+        <div class="social-buttons">
+            <div class="social-circle1">
```
Comment (apanpanpan) [inline index.html:338]: su'ad actually already made classes for these social buttons! lines 85-100 in the .scss file where she has "blue-circle" "purple-circle" etc. lets use those instead (i think the size she had for those is better too)

File: css/main.css (hunk 1 above)
Comment (apanpanpan) [reply css/main.css:166]: ahh wait yeah this is hard LOL ill take a look into it today and see if i can figure it out

File: scss/main.scss [hunk 5]
```diff
@@ -179,3 +166,45 @@ $grey: #C4C4C4;
...
+    background-color: $light-grey;
+    text-align: center;
+}
+
+.contact-content a:link {
+    color: #000;
+}
+
+//contact us circles
+.blue-circle {
```
Comment (apanpanpan) [inline scss/main.scss:191]: oop can u run the 'npm run web' again? it looks like some of ur changes didnt go to the .css file. everything else looks good tho!!

**Pull request merged**